Extension/package.json

```diff
@@ -11,7 +11,7 @@
   },
   "license": "SEE LICENSE IN LICENSE.txt",
   "engines": {
-    "vscode": "^1.57.0"
+    "vscode": "^1.58.0"
```

Comment:
Is this going to be too recent for 1.6?

Reply:
The current released version of VS Code is 1.58.2.  If we need to lag behind publicly available versions, how far do we need to lag behind?

Reply:
I'm not sure. It somewhat depends on how much we need the new feature too. Maybe ask Bob? Bob said the change to the 1.57.0 dependency was okay.